backend/src/game/cards/Card_JSON_Parser.py

```diff
@@ -0,0 +1,56 @@
+import json
+from enum import Enum
+from typing import List
+
+with open('backend\\src\\game\\cards\\KOTCards.json', "r") as json_file:
+    card_file = json.load(json_file)
+
+
+class CardType(Enum):
+    keep = 1
+    discard = 2
+    invalid = 99
+
+
+class Card(object):
+    def __init__(self, name, cost, card_type, effect, footnote):
+        self.name = name
+        self.cost = cost
+        self.effect = effect
+        self.footnote = footnote
+        if str.lower(card_type) == "keep":
+            self.card_type = CardType.keep
+        elif str.lower(card_type) == "discard":
+            self.card_type = CardType.discard
+        else:
+            self.card_type = CardType.invalid
+
+    def card_details_str(self):
```

Comment:
NIT: There has been a movement away from this type of concatenation for formatting
https://www.python.org/dev/peps/pep-3101/#string-methods

Reply:
Thanks! (does NIT stand for nitpick?)

Reply:
Yep!